docs/notebooks/Poincare Evaluation.ipynb

```diff
@@ -0,0 +1,1448 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Evaluation of Poincare Embeddings"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook demonstrates how well poincare embeddings trained using this [implementation](https://github.com/TatsuyaShirakawa/poincare-embedding) perform on the tasks detailed in the [original paper](https://arxiv.org/pdf/1705.08039.pdf).\n",
+    "\n",
+    "This is the list of tasks - \n",
+    "1. WordNet reconstruction\n",
+    "2. WordNet link prediction\n",
+    "3. Link prediction in collaboration networks\n",
+    "4. Lexical entailment on HyperLex\n",
+    "\n",
+    "A more detailed explanation of the tasks and the evaluation methodology is present in the individual evaluation subsections."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Setup\n",
+    "\n",
+    "The following code clones the `poincare-embedding` repository containing the C++ implementation of the Poincare embeddings, and applies a patch containing minor additions to the implementation. Please set the variable `parent_directory` below to define the directory to which the repository is cloned."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Set this to the path of the directory that is to contain the poincare-embedding directory\n",
+    "parent_directory = '/home/jayant/projects/'"
```

Comment:
Please use path relative to the notebook, so that you don't need to hardcode your home dir path.

Reply:
Done.